static/usage/picker/multiple-column/javascript.md

```diff
@@ -0,0 +1,79 @@
+```html
+<ion-app>
+  <ion-content fullscreen>
```

Comment:
While this technically works, `fullscreen` isn't an attribute on `ion-content`. We should have it say `fullscreen="true"`.

Reply:
Actually do we even need to use fullscreen in any of these samples?